client/public/index.html

```diff
@@ -32,8 +32,8 @@
   
     <title>Happening - Find your experience of a lifetime</title>
   </head>
-  <body>
+  <body style="height: 100vh">
```

Comment:
We could change these to className="h-screen" so that they conform with the rest of the tailwind?

Reply:
can't changed to className since it renders not as class but classname

Reply:
for some reason couldn't use class also so just used a usual style

Reply:
Aha, must be because it's a html and not a js file. Makes sense.